lte/protos/session_manager.proto

```diff
@@ -77,17 +77,32 @@ message RatSpecificContext {
   }
 }
 
+message TeidSet {
+  bytes teid = 1;
+  bytes end_ipv4_addr = 2;
```

Comment:
why teid is defined as bytes rather than uint32_t?

Reply:
It is need from AMF team ?

Reply:
Added  the same,  Raised the Issue #5679 -->https://github.com/magma/magma/issues/5679